benchmarks/accelerate_opt/benchfile.py

```diff
@@ -24,65 +40,10 @@ async def prepare(self):
             env={"MILABENCH_PREPARE_ONLY": "1"},
         )
 
-    async def run_remote_command(self, host, command):
-        prepend = ["ssh", "-l", self.config['worker_user'],
-                   "-i", "/milabench/id_milabench",
-                   "-o", "CheckHostIP=no",
-                   "-o", "StrictHostKeyChecking=no", host]
-        new_pack = self.copy({"tag": [*self.config['tag'], host]})
-        return await new_pack.execute(
-            *prepend, *command,
-            use_stdout=True,
-	)
-
-
-    def accelerate_command(self, rank):
-        nproc = len(self.config.get("devices", [])) * self.config['num_machines']
```

Comment:
I'm not sure of the expectation here. Is `num_machine` for the entire bench? In which case if we already have a list of devices, why do we need to multiply it by what seams to already be the count of them? Is there an example config I could look at?

Reply:
It is the number of devices per machine times the number of machines (nodes), yielding the total number of "processes" for the multi-gpu multi-node job. The former is filled in by the plan method, the latter is currently configured in the bench itself.

Reply:
See here for where num_machines is currently configured: https://github.com/mila-iqia/milabench/blob/master/config/base.yaml#L316

Reply:
Ah yes devices as GPUs not sure why I didn't thought of that first. Thanks!